phoenix-core/src/main/java/org/apache/phoenix/compile/CompiledOffset.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.phoenix.compile;
+
+import com.google.common.base.Optional;
+
+//pojo
```

Comment:
I think this is a good comment.  What improvement were you thinking?

Reply:
You can tell that it's a pojo without the comment being there. I meant adding a comment about what it is going to be used to represent, since that would be more useful for someone reading the code. Either way, a comment is not necessarily even needed here, but that's just my opinion.